drivers/charger/sbs_charger/sbs_charger.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2023 Cirrus Logic, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_CHARGER_SBS_CHARGER_H_
```

Comment:
why do we need an internal header? this is not shared with other modules

Reply:
ack